### ISSUE TITLE
Consolidate Series, Series Title and Series Text (for journalArticle)

### ISSUE THREAD

**Issue by rmzelle** [citation-style-language/zotero-bits#21]
_journalArticle_ currently has "Series"/**series**, "Series Title"/**seriesTitle**, and "Series Text"/**seriesText** fields, which are likely to be redundant ("Series" and "Series Title" are both mapped to **collection-title** in CSL). However, we shouldn't discard user data, so removing fields is tricky. One opportunity is the proposed addition of "Section" (ajlyon/zotero-bits#5): we could migrate data to this field (e.g. from "Series Text"), and remove the migrated field. Another option would be concatenation.

See also http://forums.zotero.org/discussion/18322/?Focus=126054#Comment_126054

**Comment by gracile-fr** [comment]
@bwiernik : yes, "Series" and "Series Title" are used for different purposes (and not the original ones, reminded by Dan), even if they're both mapped to CSL **collection-title**. As "Series Text" is not mapped, it is almost useless.

It's hard to assume anything about the content of these fields and, then, to automate their migration (just an example, I'm using Series and Serie Title for three different things - that's not wrong at all, but certainly not universal).
As a consequence, that would be useful to have batch edit/move functions in zotero when these three fields will be reshuffled. That would give the user more control, after the migration, and in this case that's probably needed (but, perhaps, I'm talking of a very small amount of people, who are in any case, able to script such a search/replace/move).
